Makefile

```diff
@@ -16,28 +16,16 @@ build-site-controller:
 
 build-controllers: build-site-controller build-service-controller
 
-docker-build: build-controllers
+docker-build:
 	${DOCKER} build -t ${SERVICE_CONTROLLER_IMAGE} -f Dockerfile.service-controller .
 	${DOCKER} build -t ${SITE_CONTROLLER_IMAGE} -f Dockerfile.site-controller .
 
 docker-push:
 	${DOCKER} push ${SERVICE_CONTROLLER_IMAGE}
 	${DOCKER} push ${SITE_CONTROLLER_IMAGE}
 
-format:
-	go fmt ./...
-
-client-mock-test:
```

Comment:
Did you intentionally remove these makefile targets? Seems unrelated to the purpose of the PR.

Reply:
No!
Fixing it now

Reply:
The faster reviewer in the far west!